AK/String.h

```diff
@@ -74,6 +76,22 @@ class String {
         return String { short_string };
     }
 
+    // Creates a new String from a single code point.
+    static constexpr String from_code_point(u32 code_point)
+    {
+        VERIFY(code_point <= 0x10ffff);
```

Comment:
Not necessarily this PR, but we have this magic number in a lot of places, would be nice to have a global name for this

Reply:
We do. Call https://github.com/SerenityOS/serenity/blob/d73a1430047facddefbd465fd2f9b3d861fe66d3/AK/CharacterTypes.h#L100 instead.